homeassistant/components/hvv_departures/config_flow.py

```diff
@@ -0,0 +1,241 @@
+"""Config flow for HVV integration."""
+import logging
+
+from pygti.auth import GTI_DEFAULT_HOST
+from pygti.exceptions import CannotConnect, InvalidAuth
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_OFFSET, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+from homeassistant.helpers import aiohttp_client
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN  # pylint:disable=unused-import
+from .hub import GTIHub
+
+_LOGGER = logging.getLogger(__name__)
+
+SCHEMA_STEP_USER = vol.Schema(
+    {
+        vol.Required(CONF_HOST, default=GTI_DEFAULT_HOST): str,
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+SCHEMA_STEP_STATION = vol.Schema({vol.Required("station"): str})
+
+SCHEMA_STEP_OPTIONS = vol.Schema(
+    {
+        vol.Required("filter"): vol.In([]),
```

Comment:
Same here. Btw why is the list empty for this config entry, is anything permitted here?

```suggestion
        vol.Required(CONF_FILTER): vol.In([]),
```

Reply:
Changed to CONF_FILTER.
It's an empty list because it should accept an array of json objects returned from the API. Should I specify the schema for that response in there?

```json
{
    "serviceID": "HHA-U:U1_HHA-U",
    "stationIDs": ["Master:10902"],
    "label": "Fuhlsbüttel Nord / Ochsenzoll / Norderstedt Mitte / Kellinghusenstraße / Ohlsdorf / Garstedt",
    "serviceName": "U1",
}
```